custom_components/husqvarna_automower/vacuum.py

```diff
@@ -27,23 +26,17 @@
 from .const import (
     DOMAIN,
     ERROR_ACTIVITIES,
-    ERROR_STATES,
     ERRORCODES,
```

Comment:
Not used

Reply:
These should be addressed in the last two commits, just wanted to point them out